device/mellanox/x86_64-mlnx_msn4800-r0/ACS-MSN4800/hwsku.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
if we have zero ports why hwsku is with all ports?

Reply:
I didn't wanted to change the hwsku.json because I know that we probably will need to change it according to the new SN4800 spec.
we may need to add new definitions for different line cards

Reply:
please align the files with zero ports as long as we dont have the full support